DigitalLearningSolutions.Data.Tests/DataServices/TutorialContentDataServiceTests/ObjectiveTests.cs

```diff
@@ -36,5 +36,69 @@ public void GetNonArchivedObjectivesBySectionAndCustomisationId_returns_objectiv
                 }
             }
         }
+
+        [Test]
+        public void GetNonArchivedCcObjectivesBySectionAndCustomisationId_returns_objectives_correctly()
+        {
+            using (new TransactionScope())
+            {
+                connection.Execute("UPDATE Tutorials SET OriginalTutorialID = 1 WHERE TutorialID = 1137");
+                connection.Execute("UPDATE CustomisationTutorials SET DiagStatus = 0 WHERE CusTutID = 704039");
+
+                // When
+                var result = tutorialContentDataService
+                    .GetNonArchivedCcObjectivesBySectionAndCustomisationId(248, 22062, false)
+                    .ToList();
+
+                // Then
+                using (new AssertionScope())
+                {
+                    result.Count.Should().Be(3);
+                    result.First().TutorialId.Should().Be(1);
+                    result.First().TutorialName.Should().Be("Manufacturers, operating systems and applications");
+                    result.First().Possible.Should().Be(4);
+                    result.First().WrongCount.Should().Be(0);
+                    result.First().MyScore.Should().Be(0);
+
+                    result.Last().TutorialId.Should().Be(1257);
+                    result.Last().TutorialName.Should().Be("Sharing information");
+                    result.Last().Possible.Should().Be(3);
+                    result.Last().WrongCount.Should().Be(0);
+                    result.Last().MyScore.Should().Be(0);
+                }
+            }
+        }
+
+        [Test]
+        public void
+            GetNonArchivedCcObjectivesBySectionAndCustomisationId_returns_objectives_correctly_with_postlearning()
+        {
+            using (new TransactionScope())
+            {
+                connection.Execute("UPDATE Tutorials SET OriginalTutorialID = 1 WHERE TutorialID = 179");
+
+                // When
+                var result = tutorialContentDataService
+                    .GetNonArchivedCcObjectivesBySectionAndCustomisationId(83, 2793, true)
+                    .ToList();

```

Comment:
Do you think it might be worth changing these two GetNonArchivedCcObjectivesBySectionAndCustomisationId to be called with the same customisationId and sectionId? Might be a bit of a pain to set up, but that way it is easier to see that the changing of the bool does causes differences in the return.